src/node_api.h

```diff
@@ -119,6 +140,73 @@ typedef struct napi_module {
   NAPI_MODULE(NODE_GYP_MODULE_NAME, NAPI_MODULE_INITIALIZER)                   \
   napi_value NAPI_MODULE_INITIALIZER(napi_env env, napi_value exports)
 
+#else
+
+#define NAPI_MODULE_XV(modname, regfunc, api_version, priv, flags)             \
+  EXTERN_C_START                                                               \
+  static napi_module _module = {                                               \
+      NAPI_MODULE_VERSION,                                                     \
+      flags,                                                                   \
+      __FILE__,                                                                \
+      regfunc,                                                                 \
+      #modname,                                                                \
+      priv,                                                                    \
+      (uintptr_t)api_version,                                                  \
+      {0},                                                                     \
+  };                                                                           \
+  NAPI_C_CTOR(_register_##modname) { napi_module_register(&_module); }         \
+  EXTERN_C_END
+
+#define NAPI_MODULE_X(modname, regfunc, priv, flags)                           \
+  NAPI_MODULE_XV(modname, regfunc, NAPI_VERSION, priv, flags)
+
+#define NAPI_CONCAT_HELPER(text1, text2) text1##text2
+#define NAPI_CONCAT(text1, text2) NAPI_CONCAT_HELPER(text1, text2)
```

Comment:
Looks like this is the renamed `NAPI_MODULE_INITIALIZER_X`. When `NAPI_EXPERIMENTAL` is defined, `NAPI_MODULE_INITIALIZER_X` is not available anymore. Is it ok for us to remove undocumented macros?

Reply:
Usually, it must be OK to remove such macros, but we cannot say if some code did not take dependency on it. I restored it.